<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QWidget" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string>QWidget {
    background-color: #000000;
}</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLineEdit" name="display">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string>QLineEdit {
    background-color: #2b2b2b;
    color: white;
    border: 2px solid #555;
    border-radius: 10px;
    padding: 10px;
}</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="button_grid">
     <property name="spacing">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_clear">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #a6a6a6;
    color: black;
    border: none;
    border-radius: 35px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #bfbfbf;
}
QPushButton:pressed {
    background-color: #8a8a8a;
}</string>
       </property>
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_toggle_sign">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #a6a6a6;
    color: black;
    border: none;
    border-radius: 35px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #bfbfbf;
}
QPushButton:pressed {
    background-color: #8a8a8a;
}</string>
       </property>
       <property name="text">
        <string>±</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn_percentage">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #a6a6a6;
    color: black;
    border: none;
    border-radius: 35px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #bfbfbf;
}
QPushButton:pressed {
    background-color: #8a8a8a;
}</string>
       </property>
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btn_divide">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #ff9500;
    color: white;
    border: none;
    border-radius: 35px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #ffad33;
}
QPushButton:pressed {
    background-color: #cc7700;
}</string>
       </property>
       <property name="text">
        <string>÷</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btn_7">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #333333;
    color: white;
    border: none;
    border-radius: 35px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #4d4d4d;
}
QPushButton:pressed {
    background-color: #1a1a1a;
}</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btn_8">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #333333;
    color: white;
    border: none;
    border-radius: 35px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #4d4d4d;
}
QPushButton:pressed {
    background-color: #1a1a1a;
}</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btn_9">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #333333;
    color: white;
    border: none;
    border-radius: 35px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #4d4d4d;
}
QPushButton:pressed {
    background-color: #1a1a1a;
}</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btn_multiply">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #ff9500;
    color: white;
    border: none;
    border-radius: 35px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #ffad33;
}
QPushButton:pressed {
    background-color: #cc7700;
}</string>
       </property>
       <property name="text">
        <string>×</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btn_4">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #333333;
    color: white;
    border: none;
    border-radius: 35px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #4d4d4d;
}
QPushButton:pressed {
    background-color: #1a1a1a;
}</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btn_5">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #333333;
    color: white;
    border: none;
    border-radius: 35px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #4d4d4d;
}
QPushButton:pressed {
    background-color: #1a1a1a;
}</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btn_6">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #333333;
    color: white;
    border: none;
    border-radius: 35px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #4d4d4d;
}
QPushButton:pressed {
    background-color: #1a1a1a;
}</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="btn_subtract">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #ff9500;
    color: white;
    border: none;
    border-radius: 35px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #ffad33;
}
QPushButton:pressed {
    background-color: #cc7700;
}</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btn_1">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #333333;
    color: white;
    border: none;
    border-radius: 35px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #4d4d4d;
}
QPushButton:pressed {
    background-color: #1a1a1a;
}</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btn_2">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #333333;
    color: white;
    border: none;
    border-radius: 35px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #4d4d4d;
}
QPushButton:pressed {
    background-color: #1a1a1a;
}</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btn_3">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #333333;
    color: white;
    border: none;
    border-radius: 35px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #4d4d4d;
}
QPushButton:pressed {
    background-color: #1a1a1a;
}</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="btn_add">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #ff9500;
    color: white;
    border: none;
    border-radius: 35px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #ffad33;
}
QPushButton:pressed {
    background-color: #cc7700;
}</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="btn_0">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #333333;
    color: white;
    border: none;
    border-radius: 35px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #4d4d4d;
}
QPushButton:pressed {
    background-color: #1a1a1a;
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="btn_decimal">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #333333;
    color: white;
    border: none;
    border-radius: 35px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #4d4d4d;
}
QPushButton:pressed {
    background-color: #1a1a1a;
}</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="btn_equals">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #ff9500;
    color: white;
    border: none;
    border-radius: 35px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #ffad33;
}
QPushButton:pressed {
    background-color: #cc7700;
}</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="btn_square">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #333333;
    color: white;
    border: none;
    border-radius: 35px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #4d4d4d;
}
QPushButton:pressed {
    background-color: #1a1a1a;
}</string>
       </property>
       <property name="text">
        <string>x²</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>